core/templates/dev/head/pages/exploration_editor/statistics_tab/StateImprovementSuggestionService.js

```diff
@@ -30,8 +30,16 @@ oppia.factory('StateImprovementSuggestionService', [
         var rankedStates = [];
         explorationStates.getStateNames().forEach(function(stateName) {
           var stateStats = allStateStats[stateName];
-          var totalEntryCount = stateStats.total_entry_count;
-          var noAnswerSubmittedCount = stateStats.no_submitted_answer_count;
+          if (ENABLE_NEW_STATS_FRAMEWORK) {
+            var totalEntryCount = (
```

Comment:
Actually, consider having a to_frontend_dict() in the backend that does the computations and returns a result which contains fields like total_hit_count (which sums the v1 and v2 counts). I'm guessing the frontend doesn't really need to know about v1 and v2, does it?

Reply:
Thats True actually. Much better way. Thanks for pointing that out.